### PR TITLE
#10264: Layer visibility limits may prevent the Info panel of search results from opening

### PR DESCRIPTION
**Pull request by mahmoudadel54**
## Description
In this PR, getting the selected search item info is implemented and showing the results into the info [identify] panel even if the layer visibility limit. 
the getFeature [identify] action is isolated from epic 'searchItemSelected' into a new epic called 'getFeatureInfoOfSelectedItem' to be applied after zoom and addMarker events  just to make sure implementing zoom action first.

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: enhancement

## Issue
#10264 

**What is the current behavior?**
#10264 

**What is the new behavior?**
If the layer has a limited visibility like showing features from a specific scale level, and user searches for an existing feature into the searched layer while the hidden scale level ---> if the feature is existing, it will zoom to this feature and opening the info panel showing the feature info data

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

File: web/client/epics/search.js [hunk 1]
```diff
@@ -213,6 +174,58 @@ export const searchItemSelected = (action$, store) =>
             return Rx.Observable.of(resultsPurge()).concat(itemSelectionStream, nestedServicesStream, searchTextStream);
         });
 
+/**
+ * Gets every `TEXT_SEARCH_ADD_MARKER` event.
+ * It is the extentsion of `TEXT_SEARCH_ITEM_SELECTED` event
+ * it resposible for retrieving feature info of the selected search results that zoom to and added a merker for by `TEXT_SEARCH_ITEM_SELECTED` event
```
Comment (dsuren1) [inline web/client/epics/search.js:180]:
```suggestion
 * Handles performing a GFI on the selected search results after zooming in, and adds a marker
```

**Review by dsuren1 — CHANGES_REQUESTED** [review]

**Review by dsuren1 — APPROVED** [review]

**Comment by dsuren1** [comment]
@ElenaGallo Kindly test it in DEV and let us know if it's good to be backported. Thanks!

**Comment by tdipisa** [comment]
@ElenaGallo please double check you test setup and let us know if the test passes.

**Comment by ElenaGallo** [comment]
Test passed, @mahmoudadel54 please backport to 2024.01.xx. Thanks

**Pull request merged**